Configuration/DataProcessing/python/Merge.py

```diff
@@ -10,6 +10,7 @@
 
 from FWCore.ParameterSet.Config import Process, EndPath
 from FWCore.ParameterSet.Modules import OutputModule, Source, Service
+from Configuration.EventContent.EventContent_cff import NANOAODEventContent
```

Comment:
this was noticed in #22933 

Some different way to load EventContent_cff has to happen here
This file uses eras and it can only work properly after the cms.Process is created.